### ISSUE TITLE
Settings dialog is not accessible

### ISSUE THREAD

**Issue by chpasha** [Darkyenus/DarkyenusTimeTracker#29]
Somehow I've managed to get the settings dialog shown once, but since then idea 2020 just shows it's popup with status bar settings every time I right click on timer

**Comment by Darkyenus** [comment]
Problems with future versions will be fixed after these versions come out - fixing against EAP versions would be a waste of time. But thank you for letting me know.

**Comment by chpasha** [comment]
fair enough, glad that you still support this plugin

**Comment by chpasha** [comment]
by the way, IDEA 2020 is RC0 now, so it's plugin API should be stable enough

**Comment by kion** [comment]
I'm also having this issue in the IDEA 2020 RC1. I'd suppose it should be pretty safe to fix against the Release Candidate versions. 

P.S. This plugin is great - thanks a lot for your work!

**Comment by Darkyenus** [comment]
Thank you for info and kind words. I guess that it will be a safe fix, but I don't have much free time right now, so I want to minimize any potential extra work that I have to do. PR or any info on what is the underlying issue is welcome.

**Comment by chpasha** [comment]
I would be glad to help, unfortunately I have no experience with idea plugins. There is no error - the only thing that I noticed is that the plugin is not visible among status bar widgets (th, so maybe it needs some kind of registration at least

**Comment by kion** [comment]
I actually do see the widget itself and it does seem to work fine in terms of tracking the time. I also am able to click on it with LMB to stop/resume time tracking. But it doesn't intercept / react to the RMB click and, thus, the settings popup is not accessible (IDEA's context menu appears instead).

**Comment by chpasha** [comment]
I mean the new menu in IDEA where you can turn on/off all status bar widgets. TimeTracker is not among them so it (indirectly) means that all status bar widgets are supposed to register themselves now to be there

**Comment by kion** [comment]
Oh, I see what you're saying. It's visible, but perhaps not "registered", so RMB click is not being passed to it by IDEA or something along those lines. Makes sense.

**Comment by kion** [comment]
I got some experience with Swing, but none with the IntelliJ Platform SDK (or didn't until now, I guess :)), but this one was pretty straightforward to fix (and the corresponding IntelliJ Platform APIs looked pretty self-explanatory by the most part). 

Note: it doesn't seem like the right mouse click is going to work anymore... Or at least I wasn't able to find a way to make it work. It looks like the right mouse click on the status bar in the IDEA 2020 is now reserved for the capability to toggle different status bar widgets (mouse event isn't even passed down to the plugin), so I had to come up with a different scheme for the options popup trigger... 

I decided to make it a bit more flexible and went with two different triggers (either one can be used):

- middle mouse button click (perhaps the best option, unless your mouse doesn't have the middle button :))
- ctrl + left mouse button click (the single-button Apple mouse users can do it too now! :) or, again, if your mouse doesn't have the middle button...)

I also made a couple of additional (rather minor) tweaks - see the pull request description for details.
